GliaWidgets/Component/Connect/ConnectView.swift

```diff
@@ -37,35 +56,39 @@ final class ConnectView: BaseView {
         )
         super.init()
     }
-   
+
     required init() {
         fatalError("init() has not been implemented")
     }
 
     override func setup() {
         super.setup()
-        setState(.none, animated: false)
         accessibilityElements = [operatorView, statusView]
 
         addSubview(stackView)
         stackView.axis = .vertical
-        stackView.spacing = 32
+        stackView.spacing = State.none.chatContentSpacing
```

Comment:
Seems like a weird name, `State.none.chatContentSpacing`. Don't really have a better one on the top of my head though. I guess it makes sense if there's state `none`, `connecting`, etc. But it's still weird that a state can be `none`.

Reply:
`none` state is used as default/initial one. What do you think about to name it `initial`?

Reply:
initial sounds nicer indeed

Reply:
You can use instead:

`private lazy var stackView = UIStackView.make(.vertical, spacing: 32)()`

Reply:
> initial sounds nicer indeed

22b5f39d04f4285c683d277b8b224c6853381dda

Reply:
> You can use instead:
> 
> `private lazy var stackView = UIStackView.make(.vertical, spacing: 32)()`

46f3f58990eb69716eef07a92e8a7ef23fa1c489